### PR TITLE
vc.js credential verify tests option compactProof true failure

### PR DESCRIPTION
**Pull request by acarnagey**
Demonstrates that when using compactProof set to true it results in two different failures on attempting to verify a MTR VC.
This PR relates to https://github.com/transmute-industries/verifiable-data/issues/112

### CODE REVIEW

File: packages/vc.js/src/__tests__/verify-options.test.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  it("expansionMap false / compactProof true - Fails to verify verifiable credential", async () => {
+    const result = await vcjs.verifiable.credential.verify({
+      credential: verifiableCredential,
+      format: ["vc"],
+      documentLoader,
+      suite: [new Ed25519Signature2018()],
+      expansionMap: false,
+      compactProof: true
+    });
+    // FIXME: https://github.com/transmute-industries/verifiable-data/issues/112
```
Comment (OR13) [inline packages/vc.js/src/__tests__/verify-options.test.ts:73]: Do we expect this to ever work? I don't think so... so the FIXME is not correct.

Instead this comment should say: "We expect this signature to fail to verify when preprocessing breaks the message payload, in this case, compactProof true is preprocessing that breaks the message payload, and as such we expect the signature to fail"... or something similar.

**Review by OR13 — CHANGES_REQUESTED** [review]
These tests are good, but their documentation is not... We don't expect these tests to ever work.... we want to be clear about that. We want to say: "we found an edge case where pre-processing broke the signature, here are some tests, these tests won't ever pass (failure is expected) and the failure is related to pre-processing the credential".

currently there are "fixme" links, but we don't actually plan to fix anything... we already have fixed this issue, the comments should be updated to explain the reason for the tests... feel free to retain the links to the issue, but only after updating the issue.

**Review by OR13 — APPROVED** [review]
